Makefile

```diff
@@ -0,0 +1,72 @@
+GO ?= go
+GOFMT ?= gofmt
+V ?=
+PKGFILES = $(shell find . \( -path ./vendor -o -path ./Godeps \) -prune \
+												-o -type f -name '*.go' -print)
+PKGFILES_notest = $(shell echo $(PKGFILES) | tr ' ' '\n' | grep -v _test.go)
+GOCYCLO ?= 15
+
+TOOLS = \
+	github.com/fzipp/gocyclo \
+	github.com/opennota/check/cmd/varcheck \
+	github.com/mendersoftware/deadcode
+
+VERSION = $(shell git describe --tags --always --dirty)
+
+LDFLAGS = \
+	-ldflags "-X main.Version=$(VERSION)"
+
+ifeq ($(V),1)
+BUILDV = -v
+endif
+
+build:
+	CGO_ENABLED=0 $(GO) build $(LDFLAGS) $(BUILDV)
+
+clean:
+	$(GO) clean
+	rm -f coverage.out coverage-tmp.out
+
+get-tools:
+	for t in $(TOOLS); do \
+		echo "-- go getting $$t"; \
+		go get -u $$t; \
+	done
+
+test:
+	$(GO) test -v ./...
+
+check:
+	echo "-- checking license header"
+	./check_license.sh
```

Comment:
Drop this, it has changed completely in https://github.com/mendersoftware/mender/pull/62, and is now covered by `go test`.